visualization/test/test-page/App.jsx

```diff
@@ -32,6 +33,7 @@ import { initializeIcons } from "@uifabric/icons";
         {label: 'irisGlobal', data: irisGlobal},
         {label: 'irisDataGlobal', data: irisDataGlobal},
         {label: 'ibmData', data: ibmData},
+        {label: 'ibmDataInconsistent', data: ibmDataInconsistent},
```

Comment:
can this be added to the automated tests as well somehow, which just validate than an error is not thrown or the appropriate error is thrown?